### PR TITLE
Reattempt of #39672

### PR DESCRIPTION
**Pull request by ManojKiranA**
Reattempt of #39672 without changing method signature

### CODE REVIEW

File: src/Illuminate/Collections/Collection.php [hunk 1]
```diff
@@ -36,11 +36,12 @@ public function __construct($items = [])
      *
      * @param  int  $from
      * @param  int  $to
+     * @param  int|float  $step
      * @return static
      */
-    public static function range($from, $to)
+    public static function range($from, $to, $step = 1)
```
Comment (driesvints) [inline src/Illuminate/Collections/Collection.php:42]: The method signature is changed again on these methods.

Comment (ManojKiranA) [reply src/Illuminate/Collections/Collection.php:42]: can I target it to 9.x

Comment (ManojKiranA) [reply src/Illuminate/Collections/Collection.php:42]: can I target it to 9.x

Comment (driesvints) [reply src/Illuminate/Collections/Collection.php:42]: You can try

**Pull request closed without merging**